test_psa_target.py

```diff
@@ -246,7 +262,7 @@ def _get_parser():
         "--mcu",
         help="Build for the given MCU",
         required=True,
-        choices=["ARM_MUSCA_B1"],
+        choices=["ARM_MUSCA_B1", "ARM_MUSCA_S1"],
```

Comment:
When referring - https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/blob/master/build_psa_compliance.py#L310, As ARM_MUSCA_S1 doesn't support FF/IPC, So please look into it. As it intends to run a binary which is not compiled.

Reply:
Issue is handled, `build_psa_compliance.py` should exit, when running with Musca-S1 and IPC.